action/protocol/staking/bucket_pool_test.go

```diff
@@ -131,14 +128,12 @@ func TestBucketPool(t *testing.T) {
 			count--
 		}
 
-		if v.commit {
-			r.NoError(csm.Commit())
-		}
-		r.Equal(v.create, pool.exist)
+		r.NoError(csm.Commit())
```

Comment:
pls keep v.commit
it will be tested that after commit, the value equals from ConstructBaseView()